WordPress.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -6,8 +6,8 @@
         "repositoryURL": "https://github.com/Automattic/ScreenObject",
         "state": {
           "branch": null,
-          "revision": "df8ee4983e674b583b546bb0640ec83428be2465",
-          "version": "0.1.0"
+          "revision": "2d65beae47cdcaaf21703ce1b321f382fe0d0730",
+          "version": "0.2.0"
```

Comment:
Updated `ScreenObject` to the latest version to take advantage of the multiple-elements init method.